pkg/k8smgmt/cluster.go

```diff
@@ -317,6 +319,9 @@ func GetNodeInfos(ctx context.Context, client ssh.Client, kconfArg string) ([]*e
 			nodeInfo.Allocatable[name] = dec
 		}
 		for res, quantity := range item.Status.Capacity {
+			if res == cloudcommon.KubernetesAMDGPUResource {
+				amdGPUCount = int(quantity.Value())
+			}
```

Comment:
Even though this is different from nvidia, might worth it to put it into getNodeGPUInfo, and let getNodeGPUInfo take `item`. That way at least the count calculation is in the same function.

Reply:
Hm, yeah, but then we'd be walking the item.Status.Capacity map twice, since we'd have to then walk it in both GetNodeInfos() and getNodeGPUInfo(). If we wanted to be picky I could just flatten out getNodeGPUInfo() into GetNodeInfos(), as gpu info is potentially spread over both the item.status and item.labels. But I think it's ok for now.